data/Kazakhstan/Assembly/sources/manual/terms.csv

```diff
@@ -1,2 +1,4 @@
 id,name,start_date,end_date,source
-5,Fifth Convocation,2012-01-15,,http://www.parlam.kz/en/mazhilis/history?id=history
+5,Fifth Convocation,2012-01-15,2016-01-20,http://www.ipu.org/parline-e/reports/2165_E.htm
+6,Fifth Convocation,2012-03-20,,http://www.ipu.org/parline-e/reports/2165_E.htm
```

Comment:
This start date is highly suspicious, as it seems to be in the wrong year. Checking with the linked IPU page, I assume you _meant_ to set it to the date of the election (2016-03-20), but in general this should be set to the start date of the term (if known), which in this case is 2016-03-25.